file_link.rb

```diff
@@ -0,0 +1,15 @@
+
+module File_link
+  
+  def self.write(name,desig,dept)
+    file_name = "employeedetails.csv"
+    CSV.open(file_name, "a") do |file|
+    file << [name,desig,dept]
+    end
+  end
+  def self.get_file_name
```

Comment:
check rubocop for file name & use of new line before each method for better readability